pipenv/project.py

```diff
@@ -51,7 +51,11 @@
 def _normalized(p):
     if p is None:
         return None
-    return normalize_drive(str(Path(p).resolve()))
+    loc = Path(p)
+    if loc.is_absolute():
+        return normalize_drive(str(loc))
+    else:
+        return normalize_drive(str(loc.resolve()))
```

Comment:
based on our findings in the other PR we've been hammering away at, can you make this second part into:
```python
    else:
        try:
            loc = loc.resolve()
        except OSError:
            loc = loc.absolute()
        return normalize_drive(str(loc))
```

In order to handle Ramdisks...

Reply:
Done